CONTRIBUTING.md

```diff
@@ -1,10 +1,20 @@
-## Contributing Guidelines
+#### Thanks, for considering to contribute to this repository!
 
-1. Fork the repo.
-2. Clone repo in your local environment.
-3. Run npm install in source directory
-4. Checkout a new feature branch
-5. Make Code changes
-6. Push the new branch.
-7. Create a pull request.
-8. Tag the issue, if the pull request is related to a particular issue
\ No newline at end of file
+if you are looking forward to start contributing to beautiful world of open source. **start it right now.**
+
+- Fork the repo
+- Clone the repo
+- Create a branch using `git checkout -b feature-branch`
+- Make the required changes
+- Run below command to verify all tests are passing
+  - `npm install` (This will install all required dependencies)
+  - `npm test`
+- Create a pull request using below commands
+  - `git add --all`
+  - `git commit -m "your commit message"`
+  - `git push origin feature-branch`
+- Go to [Repository](https://github.com/arshadkazmi42/custom-date-formatting/)
```

Comment:
Can you change this url to this
`https://github.com/arshadkazmi42/simple-date-validator/`